jobs/JEVS_GLOBAL_DET_PLOTS

```diff
@@ -44,7 +44,7 @@ export VDATE_END=${VDATE_END:-$PDYm1}
 ####################################
 export SENDECF=${SENDECF:-YES}
 export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-NO}
+export SENDDBN=${SENDDBN:-YES}
```

Comment:
@malloryprow Can you please add _export SENDMAIL=${SENDMAIL:-NO}_ into your plots J-job, just incase it's needed someday?

Reply:
Resolved in https://github.com/NOAA-EMC/EVS/pull/279/commits/e9e2f630f1ac945d51a55c3f3b9c6669fd04f125